services/harmony/test/service-image-tags.ts

```diff
@@ -894,10 +1092,93 @@ describe('Service image endpoint', async function () {
           it('returns the tag we sent', async function () {
             expect(this.res.body.tag).to.eql('foo');
           });
+
+          it('returns statusLink', async function () {
+            const link = this.res.body.statusLink;
+            statusPath = new URL(link).pathname;
+            expect(link).to.include('http://127.0.0.1:4000/service-image-tag/deployment/');
+            linkDeploymentId = getDeploymentIdFromStatusLink(link);
+            expect(linkDeploymentId).to.not.be.null;
+          });
+
+          it('reaches a terminal status without timeout', async function () {
+            const noTimeout = await waitUntilStatusChange(linkDeploymentId);
+            expect(noTimeout).to.be.true;
+          });
+
+          describe('Get service deployment status permission test', async function () {
+            describe('when a user is not in the EDL service deployers or admin groups', async function () {
+              before(async function () {
+                hookRedirect('joe');
+                this.res = await request(this.frontend).get(statusPath).use(auth({ username: 'joe' }));
+              });
+
+              after(function () {
+                delete this.res;
+              });
+
+              it('rejects the user', async function () {
+                expect(this.res.status).to.equal(403);
+              });
+
+              it('returns a meaningful error message', async function () {
+                expect(this.res.text).to.equal(userErrorMsg);
+              });
+            });
+
+            describe('when a user is in the EDL service deployers group', async function () {
+              before(async function () {
+                hookRedirect('buzz');
+                this.res = await request(this.frontend).get(statusPath).use(auth({ username: 'buzz' }));
+              });
+
+              after(function () {
+                delete this.res;
+              });
+
+              it('returns status code 200', async function () {
+                expect(this.res.status).to.equal(200);
+              });
+
+              it('returns the deployment status successful', async function () {
+                const { deploymentId, username, service, tag, status, message } = this.res.body;
+                expect(deploymentId).to.eql(linkDeploymentId);
+                expect(username).to.eql('adam');
```

Comment:
I thought the user was 'buzz'?

Reply:
This is the user who kicked off the service deployment, not the user who asks to see the deployment status. So `adam` rather than `buzz`.